polaris-react/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -16,7 +16,7 @@ import styles from './ThemeProvider.module.css';
  */
 export const themeNamesLocal = ['light', 'dark-experimental'] as const;
 
-type ThemeNameLocal = typeof themeNamesLocal[number];
+type ThemeNameLocal = (typeof themeNamesLocal)[number];
```

Comment:
Bunch of these appear to be prettier-related changes.

Do you mind if we revert these and bring them in a separate PR (they're [functionally equivalent](https://www.typescriptlang.org/play?ts=4.7.2#code/KYDwDg9gTgLgBAYwgOwM7xgC2AW2AOQEM9UAZCBQgGzgF44BtAcioEsBzTGJgGjiYAmhKAGsAtKDDAorPMhjUmAXTiFUiFOgDcAWABQ+mAE8pcACrY8RPOUo16xqRABmcLLgLFgZCtQbIAVxwAI2klXT0Aeki4WLgAPQB+QxNgc0tPG197OAAKR2AXNwzrb1tqAEp-INCocP1ouITEoA))?

Reply:
Extracted here: https://github.com/Shopify/polaris/pull/11824

Reply:
Interesting, CI fails there: https://github.com/Shopify/polaris/actions/runs/8528174905/job/23361209192?pr=11824
So it would seem that we're ending up with a different version of a key package (TypeScript? Eslint?) when linting one of these packages if using yarn instead of pnpm

Reply:
Ah damn, in that case, leave the changes in this PR. Sorry for the extra work there!